### DISCUSSION TITLE
[0.76] Does React Native support ARM32? I didn't find it on the official blog.

### DISCUSSION THREAD

**Discussion by sai819** [react-native-community/discussions-and-proposals#839] [Feedback]
I can't find a description in the following documents.
https://reactnative.dev/blog/2024/10/23/release-0.76-new-architecture

**Comment by cortinico** [comment]
Yes React Native supports the 4 default Android ABIs:

https://github.com/facebook/react-native/blob/0217d7e19cf10b6b2261ccd8e78f6bf02a55fed4/gradle.properties#L7-L10

**Reply by sai819** [reply]
Thanks. It helps me a lot.
What about other platforms, including ios, macOS and windows.
